src/gluonts/torch/model/predictor.py

```diff
@@ -102,15 +102,37 @@ def __eq__(self, that):
             that.prediction_net.state_dict(),
         )
 
-    def serialize(self, path: Path) -> None:
+    def trace(self):
+        dummy_dataset = ListDataset(
```

Comment:
A slightly better (I’m not a fan of `ListDataset` and would like to get rid of it at some point) is to use a list directly:
```
dummy_dataset = [dict(start=pd.Period(…, freq=…), target=np.zeros(1))]
```
The frequency being arbitrary here looks weird, but should not be an issue, since we’re only tracing the network and it should only any frequency information it’s configured with. (Also I don’t see how else to do it.)

Reply:
I have removed this entire dummy_input code now. The PR #2307 allows us to construct the required example input much more cleanly, so that we can directly rely on the `to_torchscript` provided by PytorchLightning.